WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift

```diff
@@ -677,15 +705,19 @@ extension OrderDetailsDataSource {
             }
 
             self.shippingLabels = shippingLabels
+            self.shippingLabelOrderItemsAggregator = ShippingLabelOrderItemsAggregator(shippingLabels: shippingLabels,
```

Comment:
Totally true! I didn't notice that this was inside a variable creation block in the first place in https://github.com/woocommerce/woocommerce-ios/pull/3188, Ernesto pointed this out and I updated it there. I'll move these two toward the top of the function.

Ideally the data assignment can be separate from `reloadSections`, and that we can be sure the data don't change between two `reloadSections` calls. I could also just get the shipping labels directly from FRC like how other order details sections, but I just didn't feel comfortable getting the data from the FRC every time. Lemme know if you think this manual assignment of shipping labels from FRC doesn't seem necessary! (I tend to over-worry about things 😅)

Reply:
Moved these two lines in: https://github.com/woocommerce/woocommerce-ios/pull/3197/commits/67190d5cb2a9810c894071a14bfce354699dc5c5

Reply:
> but I just didn't feel comfortable getting the data from the FRC every time in case the value changes without triggering reloadSections.  

I agree. I think this actually means we need to rethink the `DataSource` and `ViewModel`. We can make that reactive in the future. 🙂